samples/LiveEffect/src/main/java/com/google/oboe/samples/liveEffect/MainActivity.java

```diff
@@ -130,10 +130,8 @@ public void onClick(View v) {
         LiveEffectEngine.setDefaultStreamValues(this);
         setVolumeControlStream(AudioManager.STREAM_MUSIC);
 
-        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.R) {
-            Intent serviceIntent = new Intent(ACTION_START, null, this,
-                    DuplexStreamForegroundService.class);
-            startForegroundService(serviceIntent);
+        if (!isRecordPermissionGranted()){
+            requestRecordPermission();
```

Comment:
If we already have record permission from an earlier run, then will onRequestPermissionsResult() get called?
If not then we won't get a foreground service the second time we run the app.
We may want to add:

    } else { /* start FGS */ }